bin/exclude_tests_win.ps1

```diff
@@ -0,0 +1,34 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+
+# Helper script for excluding some tests which fail under Windows
+# see https://github.com/apache/couchdb/issues/4398
+
+$ROOT_DIR = "C:\relax\releases\3.4.2\apache-couchdb-3.4.2-RC1"

```

Comment:
It might make sense to introduce variable to capture the version number which could be then passed as a parameter externally.

Reply:
Yeah maybe, but this is at this stage a manual call of the script, so I would like to leave it as is for the moment ...

Reply:
Okay.

Reply:
added some parameters and description too